ui/webui/src/components/storage/MountPointMapping.jsx

```diff
@@ -82,6 +97,22 @@ const isDuplicateRequestField = (requests, fieldName, fieldValue) => {
     return requests.filter((request) => request[fieldName] === fieldValue).length > 1;
 };
 
+const isReformatValid = (deviceData, request, requests) => {
```

Comment:
This also needs to check whether the device is actually selected -- right now I cannot reformat `/home` when `/home/Movies` is not selected at all (which is valid choice if user is ok with everything under `/home` being removed).